app/main.js

```diff
@@ -651,7 +681,13 @@ ipcMain.on('save-setting', function (event, key, value) {
     breakPlanner.naturalBreaks(value)
   }
   settings.set(key, value)
-  settingsWin.webContents.send('renderSettings', settings.data)
+  //settingsWin.webContents.send('renderSettings', settings.data)
+  if (settingsWin) {
```

Comment:
Could you please check whether we have knowledge of what window is being used to send message (from event, or from electron api)?

I do not like doing `if` here. It's possible that someone could have settings and welcome window opened at same time and thus this would not work.

Reply:
I'll check

Reply:
Welcome window is the first window that opens up when the application starts running. So I don't think this code should create any problem.

Reply:
Let's keep for the end, when we have final wording.